### PR TITLE
Adding scm connection

### PR DESCRIPTION
**Pull request by dmnjohns**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -15,6 +15,11 @@
         <module>cli</module>
     </modules>
 
+    <scm>
+        <connection>scm:git:git@github.com:rackerlabs/attributeMapping.git</connection>
+        <tag>HEAD</tag>
```
Comment (dmnjohns) [inline pom.xml]: I'm not actually certain what setting the tag to HEAD does. This was copied from the api-checker project.

Comment (wdschei) [inline pom.xml]: I was just looking at that too.

Comment (dmnjohns) [inline pom.xml]: This was the most information I could find:
http://stackoverflow.com/questions/23718601/what-is-the-usage-of-maven-pom-xml-tag-element-inside-of-scm-when-you-are

Oddly, when Repose was using Maven, we did not set the tag.

Comment (wdschei) [inline pom.xml]: That is inline with the plugin docs:

* http://maven.apache.org/plugins/maven-scm-publish-plugin/

And the `<tag>` element doesn't seem to have any bearing when use an automated build/release system like Jenkins.

With all that said, it looks good to me.

**Pull request merged**